app/models/pro/event.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+module Pro
+  ##
+  # Events from Pro JSON API.
+  class Event < Base
+  end
```

Comment:
`JSON::API` correctly guesses that the "table name" for this model is "events" so no need to set it explicitly.